app/boards/arm/bluemicro840/bluemicro840_v1_defconfig

```diff
@@ -10,6 +10,10 @@ CONFIG_ARM_MPU=y
 # enable GPIO
 CONFIG_GPIO=y
 
+CONFIG_ADC=y
+
+CONFIG_NEWLIB_LIBC=y
```

Comment:
Can you remind me why this is needed?

Reply:
To use the math functions [here](https://github.com/zmkfirmware/zmk/pull/293/files#diff-a1f4d756abd82b30b45f6d1d469c6df584597f5b23cb1f922b8247cae4c5b86fR52), newlib is required. See: https://docs.zephyrproject.org/latest/guides/c_library.html